task-list/app/views/tasks/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Create a New Task</h1>
+
+<%= form_for @task do |f| %>
+  <table>
+    <tbody>
+      <tr>
+        <td>
+          <%= f.label "Name:" %>
```

Comment:
For readability in this case, I would put the `td` elements on the same line as your code tags since that will reduce the number of lines of code in this view significantly